main/main.ino

```diff
@@ -1319,7 +1317,10 @@ void stateMeasures() {
 #endif
 
 #if defined(ZgatewayRF) || defined(ZgatewayIR) || defined(ZgatewaySRFB) || defined(ZgatewaySRFB) || defined(ZgatewayWeatherStation)
-void storeSignalValue(SIGNAL_SIZE_UL_ULL MQTTvalue) { // Store signal values from RF, IR, SRFB or Weather stations so as to avoid duplicates
+/** 
+* Store signal values from RF, IR, SRFB or Weather stations so as to avoid duplicates
+*/
```

Comment:
normally there should be a whitespace before the `*`
```suggestion
/** 
 * Store signal values from RF, IR, SRFB or Weather stations so as to avoid duplicates
 */
```